ldpd/ldp_zebra.c

```diff
@@ -506,12 +506,14 @@ ldp_zebra_connected(struct zclient *zclient)
 	    ZEBRA_ROUTE_ALL, 0, VRF_DEFAULT);
 }
 
+extern struct zebra_privs_t ldpe_privs;
+
 void
 ldp_zebra_init(struct thread_master *master)
 {
 	/* Set default values. */
 	zclient = zclient_new(master);
-	zclient_init(zclient, ZEBRA_ROUTE_LDP, 0);
+	zclient_init(zclient, ZEBRA_ROUTE_LDP, 0, &ldpe_privs);
```

Comment:
ldpe is one of the child processes (the LDP engine), so `ldpe_privs` is uninitialized here. You want to use `ldpd_privs` instead.